demo/demo.html

```diff
@@ -1,12 +1,10 @@
 <head>
     <title>GeoTIFFTileSource Demo</title>
-    <!-- <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/openseadragon/3.1.0/openseadragon.min.js"></script> -->
-    <!-- <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/geotiff"></script> -->
-    <script type="text/javascript" src="../lib/openseadragon.js"></script>
-    <script type="text/javascript" src="../lib/geotiff.js"></script> 
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/openseadragon/3.0.0/openseadragon.min.js"></script>
+
     <link rel="stylesheet" href="./demo.css">
-    
-    <script type="text/javascript" src="../GeoTIFFTileSource.js"></script>
+
+    <script type="text/javascript" src="../dist/GeoTIFFTileSource.umd.js"></script>
```

Comment:
I feel like we can use plain `.js` to indicate the UMD file, and `.mjs` for the module files. Then we wouldn't need `.umd.js` which is kind of non-standard.